crawler/http/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Repo name: arynai/sycamore-crawler-http
 
-FROM python:3
+FROM python:3.11
```

Comment:
Good point.  Since PyTorch isn't compatible with 3.11 and the UI specifies 3.11, this seemed the saner route, and it requires one less Docker image on my laptop.